src/components/CardView/CardView.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+const CardView = (props) => {
+  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+
+  let imageEle;
+  if (imageUrl) {
+    imageEle = <img className="card-img-top" src={imageUrl} alt={imageAlt} />;
+  }
+
+  let buttonEle;
+  if (buttonUrl && buttonText) {
+    buttonEle = <a href={buttonUrl} className="btn btn-secondary" target="_blank">{buttonText}</a>;
```

Comment:
can usage of link for a button be as accessible as using `<button>`?
Will you be able to disable it?